src/parcsr_ls/par_relax_device.c

```diff
@@ -123,7 +123,8 @@ hypre_BoomerAMGRelaxTwoStageGaussSeidelDevice ( hypre_ParCSRMatrix *A,
    /* Sanity checks */
    hypre_assert(u_vecstride == num_rows);
    hypre_assert(r_vecstride == num_rows);
-   hypre_assert(z_vecstride == num_rows);
+   /* WM: TODO - commenting out the assert below, which fails */
```

Comment:
We can remove the TODO here and the assertion itself if you want, I have a branch fixing the assert failure coming soon

Reply:
Done. I also remove the `z_vecstride` variable to avoid a compiler warning about it non being used.